config/default.yaml

```diff
@@ -36,3 +37,21 @@ modules:
     # - ./modules/statsd
     # - ./modules/openTSDB
     # - ./modules/influxdb
+
+  whitelistedKeys:
+  # Uncomment the keys that you'd like to whitelist
```

Comment:
A note here would be helpful. "The example below are timing metrics that Bucky includes by default. You will need to whitelist these if you use the default configuration of Bucky and enable whitelisting"